### PR TITLE
Add gradients feature

### PR DESCRIPTION
**Pull request by jamesnw**
I was on the fence about combining linear and radial gradients, but this mirrors Can I Use and MDN, and there isn't a separate support story.

### CODE REVIEW

**Comment by vwallen** [comment]
Splitting `linear`/`repeating-linear` and `radial`/`repeating-radial` is more consistent with the `conic-gradients` feature, and makes sense within the `gradients` group

File: features/gradients.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Gradients
+description: The `linear-gradient()`, `radial-gradient()` CSS functions and their repeating counterparts create background that progress smoothly between multiple colors.
```
Comment (ddbeck) [inline features/gradients.yml:2]: nits

```suggestion
description: The `linear-gradient()` and `radial-gradient()` CSS functions and their repeating counterparts create backgrounds that progress smoothly between multiple colors.
```

File: features/gradients.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+spec: https://drafts.csswg.org/css-images-3/#gradients
+# Todo: caniuse support needs to be reconciled for Safari and Firefox.
```
Comment (ddbeck) [inline features/gradients.yml:4]:
```suggestion
# TODO: caniuse support needs to be reconciled for Safari and Firefox.
```

**Review by ddbeck — APPROVED** [review]
This is good, but I have a couple extremely minor textual issues to fix.

**Comment by ddbeck** [comment]
Oh, on the question of splitting: I'm agnostic. If you want to split it, I'll re-review. But I suspect we'll want a way to address both and a `TODO` saying something like "we should have separate linear and radial grandients when https://github.com/web-platform-dx/web-features/issues/971 is resolved" would also be fine.

**Pull request merged**